src/Microsoft.Data.SqlClient/netfx/ref/Microsoft.Data.SqlClient.cs

```diff
@@ -18,7 +20,19 @@ public sealed partial class OperationAbortedException : System.SystemException
         private OperationAbortedException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
 
     }
+
+    /// <summary>
+    /// Extensions for SqlDbType enum to enable its usage.
+    /// </summary>
+    public static class SqlDbTypeExtensions
```

Comment:
Why not include docs from the same XML here too?

Reply:
That's a miss. Thanks for catching

Reply:
Fixed